@keys: Improve documentation about logical keys (#10963)
slint-ui/slint: docs/astro/src/content/docs/reference/keyboard-input/overview.mdx

### Different Keyboard Layouts

> Using the `@keys(..)` macro with a key from the [`Key` namespace](#key-namespace) will handle different keyboard layouts automatically.

Reviewer: ```suggestion
Different keyboard layouts are handled automatically by the `@keys(..)` macro for keys from the [`Key` namespace](#key-namespace).
```

For example, `@keys(Control + Plus)` fires whenever the user presses the key that produces `+` on their layout — on a US layout that requires `Shift` + `=`, while on a German layout `+` has its own key and no `Shift` is needed.

Named keys inside `@keys(..)` are logical: `A` represents the character `a`, not a specific physical key location. This means `@keys(Control + A)` will activate on whichever key produces `a` on the user's layout, regardless of where that key sits on the keyboard.

> If the [`Key` namespace](#key-namespace) does not include the needed key, the @key macro also supports advanced keyboard shortcut definitions...

Author: reworded
